ingestion-lambda/src/handler.ts

```diff
@@ -18,6 +18,22 @@ interface OperationSuccess {
 
 type OperationResult = OperationFailure | OperationSuccess;
 
+const isCurlyQuoteFailure = (e: SyntaxError): boolean => {
+	return !!e.message.match(/Unexpected token '[“‘”’]'/);
+};
+
+const safeBodyParse = (body: string): any => {
+	try {
+		return JSON.parse(body);
+	} catch (e) {
+		if (e instanceof SyntaxError && isCurlyQuoteFailure(e)) {
+			console.warn('Stripping badly escaped curly quote');
+			return JSON.parse(body.replaceAll(/\\([“‘”’])/g, '$1'));
```

Comment:
I think this might cause issues if there's an escaped backslash before a quote?

Maybe we want to make sure there isn't a `\\` in front of the quotes? e.g. `(^\\|[^\\]\\)[“‘”’]`.

https://regex101.com/r/r0V6ab/1

Reply:
good point! I was feeling lazy, as literal backslashes are unlikely to appear in prose, especially near curly quotes...
Plus if we want to follow that rabbit hole, we then would need to consider if there was an escaped backslash and _then_ a badly-escaped curly quote, ie. `\\\“` should become `\\“`...

I think this is negative lookbehind territory, I got to https://regex101.com/r/6n6oNh/1 which seems to be behaving fairly well but looks pretty monstrous

Reply:
It's a fair point, maybe this concern is just WONTFIX.

Reply:
added a comment explaining why this is only a partial fix, and some next steps if the problem rears its head again